src/components/Modals/components/FacebookAuthentication.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <v-facebook-login @sdk-init="handleSdkInit" :app-id="fbID" @login="fbLogIn()"></v-facebook-login>
+</template>
+
+<script lang="ts">
+  import { Component, Prop, Vue } from 'vue-property-decorator';
+
+  // TODO https://github.com/eternagame/eternagame.org/issues/17 improve typing
+  // @ts-ignore
+  import VFacebookLogin from 'vue-facebook-login-component';
+
+  @Component({
+    components: {
+      VFacebookLogin,
+    },
+  })
+  export default class FacebookAuthentication extends Vue {
+    private FB = null;
+
+    private fbID = process.env.VUE_APP_FACEBOOK_API_ID;
+
+    async fbLogIn() {
+      const data = await this.$vxm.user.fbLogin(this.FB);
+      this.$emit('fb-verify', data);
+    }
+
+    // TODO https://github.com/eternagame/eternagame.org/issues/17 improve typing
+    // @ts-ignore
+    handleSdkInit({ FB }) {
```

Comment:
I think that `({ FB }: any)` would be better than `@ts-ignore`, but I'm not sure.

Reply:
Use any instead of ts-ignore - if we ts-ignore, it will prevent typescript from reporting any other errors.

Reply:
changed to :any in many instances. other instances are npm components that don't support typescript apparently